core/tests/gae_suite.py

```diff
@@ -22,10 +22,10 @@
 from the oppia/ root folder.
 """
 
-import argparse
 import os
 import sys
 import unittest
+import argparse
```

Comment:
Actually, it's not worth spending a lot of time on this one, I think. For the 'argparse' imports (and only those), I would add a ```# pylint: disable``` pragma (see elsewhere in the code for how this is done).